unix-ffi/json/json/__init__.py

```diff
@@ -413,4 +413,6 @@ def loads(
         kw["parse_int"] = parse_int
     if parse_constant is not None:
         kw["parse_constant"] = parse_constant
+    if isinstance(s, bytes):
```

Comment:
CPython also supports `bytearray` so this could be:
```
if isinstance(s, (bytes, bytearray)):
```

Also, please also update the documentation comment for the `loads` function to mention that `s` can be str, bytes or bytearray.

Reply:
Updated.